argilla/docs/how_to_guides/export.md

```diff
@@ -0,0 +1,156 @@
+---
+description: In this section, we will provide a step-by-step guide to show how to filter and query a dataset.
+---
+
+# Exporting from Argilla
+
+This guide provides an overview of how to export your dataset or its records to Python, your local disk, or the Hugging Face Hub.
+
+In Argilla, you can export two main components of a dataset:
+- The dataset's complete configuration defined in `rg.Settings`. This is useful if your want to share your feedback task or restore it later in Argilla.
+- The records stored in the dataset, including `Metadata`, `Vectors`, `Suggestions`, and `Responses`. This is useful if you want to use your dataset's records outside of Argilla.
+
```

Comment:
Should we add a general redirect to API references at the top of the page as in other guides? https://argilla-io.github.io/argilla/dev/how_to_guides/workspace/

Reply:
Also, I think you might need to update the API reference somewhere?

Reply:
I've added a reference to `rg.Dataset`. There is not a single API reference for this guide because it uses both `Dataset` and `Dataset.records`.